src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import duke.exception.CommandInvalidException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDos;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String path;
+    private static final DateTimeFormatter DATE_FORMAT_INP = DateTimeFormatter.ofPattern("yyyy-MM-dd' 'HH:mm");
+    private static final DateTimeFormatter DATE_FORMAT_OUT = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+    private File data;
+
+    /**
+     * Constructor for storage
+     *
+     * @param filePath path of file in string.
+     */
+    public Storage(String filePath) {
+        path = filePath;
+        data = new File(path);
+    }
+
+    /**
+     * Returns an ArrayList of task from the saved file.
+     *
+     * @return an ArrayList contains Task.
+     * @throws FileNotFoundException If file is not yet made.
+     */
+    public ArrayList<Task> load() throws FileNotFoundException {
+        ArrayList<Task> taskF = new ArrayList<>();
```

Comment:
Perhaps change the name is this variable to tasks? Your list contains multiple tasks so its name should be plural.

Reply:
Oh yea, thanks for noticing it!